packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -482,7 +482,8 @@ class SliderThemeData with Diagnosticable {
   /// [Slider] is disabled.
   final Color disabledThumbColor;
 
-  /// The color of the overlay drawn around the slider thumb when it is pressed.
+  /// The color of the overlay drawn around the slider thumb when it is
+  /// pressed, focused, or hovered.
```

Comment:
Most material design elements have different colors for hover and focus, since you want to be able to see the hover when it's already focused, otherwise there's no visible feedback.

Reply:
This is the material spec from https://material.io/components/sliders/
![unnamed](https://user-images.githubusercontent.com/591699/78685183-6530b900-78bf-11ea-963a-c65cdf44778b.png)

Reply:
OK, I guess if that's the spec.  `¯\_(ツ)_/¯`